run-test.py

```diff
@@ -57,6 +59,42 @@
 
 CURRENT_TEST_NAME = ""
 
+K8S_YAML_TOOLS_TEMPLATE = """
+kind: Service
+apiVersion: v1
+metadata:
+  name: jepsen-100
+spec:
+  type: NodePort
+  ports:
+    - port: 22
+      nodePort: 32100
+  selector:
+    name: jepsen-100
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: jepsen-100
+  labels:
+    name: jepsen-100
+    app: insolar-jepsen
+spec:
+  containers:
+    - name: jepsen-100
+      image: {image_name}
+      imagePullPolicy: {pull_policy}
+      securityContext:
+        capabilities:
+          add:
+            - NET_ADMIN
+      ports:
+        - containerPort: 22
+  nodeSelector:
+    jepsen: "true"
+---
+"""
```

Comment:
I'm not convinced that running a separate pod for just sending HTTP requests to other pods is such a good idea. We already have quite a lot of pods. I suggest to rewrite it and send all requests e.g. from heavy, as we currently do with benchmark, pulsewatcher, etc.

Reply:
I suggest to use another pod for all this actions, collecting metrics is just a start

Reply:
Why / what for? What this will accomplish?

Reply:
Because it will be more transparent and convenient, and in code too

Reply:
Personally I'm not convinced. However if, say, @LinnikD and @megge-dream OK with it I'm OK too.